pkg/controllers/clusters/clusterstatus/clusterstatus_controller.go

```diff
@@ -191,7 +191,7 @@ func (c *Controller) GetManagedClusterLabels() labels.Set {
 		klog.Warningf("failed to list nodes: %v", err)
 		return nil
 	}
-	return getCommonNodeLabels(nodes)
+	return aggregateLimitedLabels(nodes, 0.8)
```

Comment:
Could we add a flag for this setting on agent side?

Reply:
yes, i intended to add a flag in another pr. Now i put them together. please take another look.